slynk/slynk-loader.lisp

```diff
@@ -136,7 +136,7 @@ operating system, and hardware architecture."
 (defun sly-version-string ()
   "Return a string identifying the SLY version.
 Return nil if nothing appropriate is available."
-  (let ((this-file #.(or *compile-file-truename* *load-truename*)))
+  (let ((this-file #.(or #-clasp *compile-file-truename* *load-truename*)))
```

Comment:
OK.  I'm reading the explanation in the commit message and I think I get it.  But that message should be even clearer.  if this rewording is correct, use something like it instead :

```
 Clasp *compile-file-truename* will be /tmp

Clasp will set *compile-file-truename* to /tmp/something.faslclasp
the containing directory of which likely won't contain 'sly.el'.  However
*load-truename* will contain the path to slynk-loader.lisp, and 'sly.el'
should be found as a result.
```

Also, as far as I remember loading `slynk-loader.lisp` is just another "legacy" way of loading SLY.  The preferred way is to load it via ASDF.  Does this work with Clasp? Did you test it?

Reply:
Ok, how would I change the commit message?  Or can you change it.   Your version is more readable.

Reply:
Reworded as per @joaotavora.